docs/usage.md

```diff
@@ -69,119 +115,360 @@ cat ./compilation.json
+CodeChecker parse --print-steps ./reports-ctu
+```
+
+The example code have an other bug!
+
+```sh
+[HIGH] .../docs/examples/src/divide.c:5:22: Division by zero [core.DivideZero]
+    return numerator / denominator;
+                     ^
+  Report hash: 0865bada8cc2c49d99f981d8c6484bd7
+  Steps:
+    1, main.c:17:44: Passing the value 0 via 2nd parameter 'denominator'
+    2, main.c:17:26: Calling 'divide'
+    3, divide.c:3:1: Entered call from 'main'
+    4, divide.c:5:22: Division by zero
+
+Found 1 defect(s) in divide.c
+
+```
+
+## Step 6: Store analysis results in a CodeChecker DB and visualize results<a name="step-6"></a>
+You can store the analysis results in a central database and view the results
+in a web viewer:
+
+1. Start the CodeChecker server locally on port 8555 (using SQLite DB, which is
+not recommended for multi-user central deployment) create a workspace
+directory, where the database will be stored.
+
+Note: Use a new shell.
+
+```sh
+cd <repo root dir>
+mkdir ./ws
+CodeChecker server --workspace ./ws -v 8555
+```
+A default product called `Default` will be automatically created where you can
+store your results.
+
+2. Store the results in the server under name "example" (in the `Default`
+product):
+
+```sh
+CodeChecker store ./reports-ctu --name example \
+    --url http://localhost:8555/Default
+```
+
+The URL is in `PRODUCT_URL` format:
+`[http[s]://]host:port/ProductEndpoint`
+Please note that if you start the server in secure mode (with SSL) you will
+need to use the `https` protocol prefix. The default protocol is `http`.
+See [user guide](web/user_guide.md#product_url-format) for detailed
+description of the `PRODUCT_URL` format.
+
+3. View the results in your web browser
+```sh
+firefox http://localhost:8555/Default &
+```
+
+### Definition of "run"<a name="run-definition"></a>
+A run, in the simplest case is a single analysis snapshot of your software.
+
+You can follow up the quality status of your product by storing the analysis
+results of consecutive git commits of a git branch into the same run. In this
+case, you will be able to follow-up the outstanding results and fixed reports
+in the statistics views. (
+[See your first example run](http://localhost:8555/Default/runs))
+
+So a run represents the analysis status of a single branch of your code, with a given analysis configuration. You can also record the source code version associated with the analysis with the `--tag` parameter.
+
+Make sure, that you use the same analysis configuration when updating a run,
+because a changed analysis configuration can make new reports to appear or
+reports to disappear.
+
+### Programmer checking new bugs in the code after local edit (and compare it to a central database)<a name="compare"></a>
+Say that you made some local changes in your code (automatically fixing
+example program) and you wonder whether you fixed any bugs. Each bug has a
+unique hash identifier that is independent of the line number, therefore
```

Comment:
```suggestion
[unique hash identifier](analyzer/report_identification.md) that is independent of the line number, therefore
```

Reply:
Done.

Reply:
It is marked as resolved but it is still not resolved.